scripts/train.py

```diff
@@ -1,8 +1,11 @@
+import os
 import importlib
+os.environ['TORCH_CUDNN_V8_API_ENABLED'] = '1' # Prevent unacceptable slowdowns when using 16 precision
 
 from utils.hparams import set_hparams, hparams
 
-set_hparams(print_hparams=False)
+set_hparams()
+hparams['disable_sample_tqdm'] = True
```

Comment:
Also, is it possible moving this assignment into hparams.py, before calling `print_hparams()`, and make sure it only happens when training? When running inference from infer.py, `hparams['infer']` will be set to `True`.